### PR TITLE
new icon: centos (original, original-wordmark)

### PR DESCRIPTION
**Pull request by MichielPater**

### CODE REVIEW

**Review by amacado — CHANGES_REQUESTED** [review]
Hello @MichielPater and thank you for the contribution. When I took a look at the icons I noticed that the `original-wordmark` doesn't look right:
image

The `original` looks better, but maybe you could reduce the "blank space" around it a little so it's not taking up so much empty space:
image

**Comment by Thomas-Boi** [comment]
Hey @MichielPater,

Please hold off fixing the `fill` issue reported by the bot. I'm trying to find a way to fix this issue permanently using our SVG-optimizer bot.

File: icons/centos/centos-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...fill="#232375"/><path d="M20.285 65.973l1.191-1.18 1.195 1.18v8.336h3.973l-5.168 5.078-5.082-5.078h3.891zm0 0" fill="#9ccc28"/></g><path d="M10.133 52.188h10.262v10.125H10.133zm0 0" fill="#9ccc28"/><path d="M9.766 51.827v.362 10.487h10.995V51.827zm.733.723h9.529v9.402h-9.529z" fill="#fff"/><path d="M22.594 52.188h10.266v10.125H22.594zm0 0" fill="#912178"/><path d="M22.227 51.827v.362 10.487h10.999V51.827zm.733.723h9.533v9.402H22.96z" fill="#fff"/><path d="M22.594 64.481h10.266v10.125H22.594zm0 0" fill="#eea623"/><path d="M22.227 64.12v.362 10.486h10.999V64.12zm.733.723h9.533v9.402H22.96z" fill="#fff"/><path d="M10.133 64.481h10.262v10.125H10.133zm0 0" fill="#232375"/><g fill="#fff"><path d="M9.766 64.12v.362 10.486h10.995V64.12zm.733.723h9.529v9.402h-9.529z"/><path d="M4.907 63.399l.259.256 7.517 7.416 7.776-7.672-7.776-7.671zm1.037 0l6.739-6.649 6.739 6.649-6.739 6.649zm7.776-8.695l.259.256 7.517 7.416 7.776-7.672-7.776-7.672zm1.037 0l6.74-6.649 6.739 6.649-6.739 6.649zm15.552 1.023l-7.776 7.667.259.256 7.517 7.42 7.776-7.676zm0 1.023l6.74 6.645-6.739 6.652-6.74-6.653zM13.72 72.09l.259.256 7.517 7.416 7.776-7.672-7.776-7.668zm1.037 0l6.739-6.645 6.74 6.645-6.74 6.649z"/></g></svg>
```
Comment (Panquesito7) [inline icons/centos/centos-original-wordmark.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M55.789 71.121h-6.703a7.39 7.39 0 0 1-2.016-.285c-1.289-.406-2.309-1.332-3.059-2.777-.68-1.316-1.02-2.848-1.02-4.598 0-1.687.301-3.18.91-4.473.664-1.441 1.613-2.41 2.84-2.937.957-.402 1.934-.605 2.93-.605h6.074v3h-6.312c-1.027 0-1.773.578-2.234 1.723-.363.898-.543 2.074-.543 3.52 0 1.227.23 2.262.695 3.109.535 1.004 1.285 1.508 2.254 1.508l6.184-.023v2.84m11.164.07H61.66c-1.289 0-2.41-.566-3.363-1.703-.941-1.148-1.41-2.473-1.41-3.961 0-1.629.512-3.035 1.539-4.211 1.043-1.199 2.199-1.793 3.473-1.793h5.031v2.395h-3.687c-.461 0-.859.043-1.191.137-.32.074-.617.23-.891.469-.508.449-.758 1.051-.758 1.813h6.504v2.422h-6.484c0 .355.035.668.109.914.086.242.254.492.5.742.246.238.535.402.867.492.348.078.766.113 1.258.113h3.797v2.172m12.367-.07h-3.016v-7.637c0-.187-.043-.363-.129-.512a1.72 1.72 0 0 0-.371-.473c-.289-.25-.621-.379-.996-.379h-2.949v9h-3.016V59.523h5.77c1.027 0 1.773.059 2.234.18.723.18 1.301.57 1.734 1.188a4.07 4.07 0 0 1 .738 2.352v7.879m8.66-9.18h-2.93v5.266c0 .473.16.871.477 1.184.363.313.84.473 1.434.473h.891v2.258h-2.801a2.82 2.82 0 0 1-2.145-.941c-.348-.367-.574-.781-.676-1.227-.102-.453-.152-.98-.152-1.59v-5.422h-1.906v-2.418h1.906v-4.078h2.973v4.078h2.93v2.418m17.699 1.324c0 3.043-.816 5.219-2.449 6.539-1.332 1.059-3.27 1.59-5.813 1.59-2.82 0-4.902-.609-6.25-1.84-1.473-1.34-2.211-3.5-2.211-6.469 0-2.434.656-4.332 1.973-5.687 1.434-1.484 3.574-2.223 6.422-2.223 3.137 0 5.352.707 6.637 2.133 1.129 1.25 1.691 3.238 1.691 5.957m-3.859-.09c0-1.539-.355-2.766-1.062-3.676-.797-1.02-1.977-1.523-3.535-1.523-1.535 0-2.691.543-3.473 1.637-.648.91-.977 2.098-.977 3.563 0 1.688.262 2.93.781 3.734.723 1.125 2.063 1.68 4.016 1.68 1.648 0 2.805-.539 3.469-1.633.52-.852.781-2.109.781-3.781m21.223 3.09c0 1.367-.441 2.527-1.324 3.473s-2 1.406-3.359 1.406l-10.738-.023v-2.816h8.852c.59 0 1.09-.141 1.496-.43.461-.328.691-.773.691-1.34 0-1.332-.91-1.996-2.73-1.996h-4.469c-1.289 0-2.414-.406-3.383-1.227-.969-.828-1.453-1.871-1.453-3.145 0-1.398.469-2.539 1.41-3.402.953-.883 2.203-1.32 3.75-1.32h10.262v2.691H113c-.406 0-.816.176-1.234.535-.406.344-.609.75-.609 1.234a1.81 1.81 0 0 0 .586 1.363c.406.363.891.535 1.453.535h4.926c1.531 0 2.73.414 3.598 1.234.883.805 1.324 1.883 1.324 3.227"/><g fill-rule="evenodd"><path d="M18.883 62.203l1.199 1.176-1.199 1.176h-8.449v3.922l-5.152-5.098 5.152-5.012v3.836zm0 0" fill="#912178"/><path d="M22.707 60.82L21.516 62l-1.195-1.18v-8.336h-3.977l5.172-5.078 5.078 5.078h-3.887zm0 0" fill="#eea623"/><path d="M24.109 64.594l-1.199-1.18 1.199-1.176h8.445v-3.922l5.152 5.098-5.152 5.016v-3.836zm0 0" fill="#232375"/><path d="M20.285 65.973l1.191-1.18 1.195 1.18v8.336h3.973l-5.168 5.078-5.082-5.078h3.891zm0 0" fill="#9ccc28"/></g><path d="M10.133 52.188h10.262v10.125H10.133zm0 0" fill="#9ccc28"/><path d="M9.766 51.827v.362 10.487h10.995V51.827zm.733.723h9.529v9.402h-9.529z" fill="#fff"/><path d="M22.594 52.188h10.266v10.125H22.594zm0 0" fill="#912178"/><path d="M22.227 51.827v.362 10.487h10.999V51.827zm.733.723h9.533v9.402H22.96z" fill="#fff"/><path d="M22.594 64.481h10.266v10.125H22.594zm0 0" fill="#eea623"/><path d="M22.227 64.12v.362 10.486h10.999V64.12zm.733.723h9.533v9.402H22.96z" fill="#fff"/><path d="M10.133 64.481h10.262v10.125H10.133zm0 0" fill="#232375"/><g fill="#fff"><path d="M9.766 64.12v.362 10.486h10.995V64.12zm.733.723h9.529v9.402h-9.529z"/><path d="M4.907 63.399l.259.256 7.517 7.416 7.776-7.672-7.776-7.671zm1.037 0l6.739-6.649 6.739 6.649-6.739 6.649zm7.776-8.695l.259.256 7.517 7.416 7.776-7.672-7.776-7.672zm1.037 0l6.74-6.649 6.739 6.649-6.739 6.649zm15.552 1.023l-7.776 7.667.259.256 7.517 7.42 7.776-7.676zm0 1.023l6.74 6.645-6.739 6.652-6.74-6.653zM13.72 72.09l.259.256 7.517 7.416 7.776-7.672-7.776-7.668zm1.037 0l6.739-6.645 6.74 6.645-6.74 6.649z"/></g></svg>

```

File: icons/centos/centos-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...l16.89-17.229-16.66-16.973V17.454H81.392zm-.005 3.9l12.841 13.089h-8.71v33.645l-4.13 4.206-4.175-4.257v-33.82h-8.445zm-42.26 15.633h23.107l-9.661 9.841 21.92 22.356v3.418h-4.105L31.723 33.653 22.458 42.9zm26.937 0h8.43v28.277L39.743 29.836zm22.177.215h8.695l9.431 9.626L71.569 48.32zm12.529 0h22.697v22.951l-9.436-9.62-21.856 22.291h-3.934v-3.599l21.961-22.398zm13.261 17.245l9.436 9.615v8.761H79.339zm-65.633.067l18.117 18.089H22.457V46.78zM19.74 49.487v8.878h32.864l4.49 4.472-4.03 4.027H19.739v9.328L6.364 62.837zm89.776.358l12.825 13.074-13.057 13.315v-9.374H76.246l-3.86-3.94 4.249-4.329h32.878zm-48.974 5.117l4.175 4.247 4.13-4.201v3.581h3.955l-4.249 4.329 3.86 3.94h-3.7v4.222l-4.346-4.339-3.819 3.817v-3.694H56.94l4.03-4.027-.966-.956-3.524-3.516h4.065zm-38.08 14.656H50.3L31.728 88.161l-9.265-9.256zm31.712 0h3.65v3.659L35.127 95.928l9.546 9.527H22.462V82.777l9.261 9.247zm17.256 0h3.689l22.241 22.68.961-.967 8.24-8.413v22.546H84.06l9.551-9.533-.966-.972L71.43 73.795zm7.519 0h27.613v9.39l-9.201 9.379zm-14.577.998l4.34 4.335v33.267h8.716l-13.055 13.028-13.055-13.028h9.231V74.431zm-6.547 6.534v28.307h-9.27l-9.546-9.527zm13.606.512l18.306 18.268-9.551 9.533H71.43zm0 0" fill="#fff"/></svg>
```
Comment (Panquesito7) [inline icons/centos/centos-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><style><![CDATA[.I{fill:rgb(57.647059%,13.333333%,47.45098%)}.J{fill:rgb(93.72549%,65.490196%,14.117647%)}.K{fill:rgb(61.176471%,80.392157%,16.470588%)}.L{fill:rgb(14.901961%,14.509804%,46.666667%)}]]></style><path d="M59.033 64.192l-27.305 26.66-27.302-26.66 27.302-26.651zm0 0" class="I"/><path d="M64.72 58.736L37.824 31.91 64.72 5.084 91.61 31.91zm0 0" class="J"/><path d="M91.675 96.563l-27.307 26.661-27.3-26.661L64.37 69.913zm0 0" class="K"/><path d="M97.362 91.108L70.466 64.277l26.895-26.821 26.895 26.821zm0 0" class="L"/><path d="M70.066 69.484h36.461v36.401H70.066zm0 0" class="J"/><path d="M70.216 22.454h36.577v36.231H70.216zm0 0" class="I"/><path d="M21.1 69.484h38.084v37.753H21.1zm0 0" class="L"/><path d="M21.1 20.928h38.084v37.538H21.1zm0 0" class="K"/><path d="M64.722.464L48.268 17.227H19.739v28.395L2.483 62.837l17.256 17.229v28.149h27.697l16.93 16.901 16.935-16.901h27.983V80.148l16.89-17.229-16.66-16.973V17.454H81.392zm-.005 3.9l12.841 13.089h-8.71v33.645l-4.13 4.206-4.175-4.257v-33.82h-8.445zm-42.26 15.633h23.107l-9.661 9.841 21.92 22.356v3.418h-4.105L31.723 33.653 22.458 42.9zm26.937 0h8.43v28.277L39.743 29.836zm22.177.215h8.695l9.431 9.626L71.569 48.32zm12.529 0h22.697v22.951l-9.436-9.62-21.856 22.291h-3.934v-3.599l21.961-22.398zm13.261 17.245l9.436 9.615v8.761H79.339zm-65.633.067l18.117 18.089H22.457V46.78zM19.74 49.487v8.878h32.864l4.49 4.472-4.03 4.027H19.739v9.328L6.364 62.837zm89.776.358l12.825 13.074-13.057 13.315v-9.374H76.246l-3.86-3.94 4.249-4.329h32.878zm-48.974 5.117l4.175 4.247 4.13-4.201v3.581h3.955l-4.249 4.329 3.86 3.94h-3.7v4.222l-4.346-4.339-3.819 3.817v-3.694H56.94l4.03-4.027-.966-.956-3.524-3.516h4.065zm-38.08 14.656H50.3L31.728 88.161l-9.265-9.256zm31.712 0h3.65v3.659L35.127 95.928l9.546 9.527H22.462V82.777l9.261 9.247zm17.256 0h3.689l22.241 22.68.961-.967 8.24-8.413v22.546H84.06l9.551-9.533-.966-.972L71.43 73.795zm7.519 0h27.613v9.39l-9.201 9.379zm-14.577.998l4.34 4.335v33.267h8.716l-13.055 13.028-13.055-13.028h9.231V74.431zm-6.547 6.534v28.307h-9.27l-9.546-9.527zm13.606.512l18.306 18.268-9.551 9.533H71.43zm0 0" fill="#fff"/></svg>

```

File: devicon.json [hunk 3]
```diff
@@ -515,6 +515,24 @@
...
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#932178",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:535]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 4]
```diff
@@ -516,6 +516,33 @@
...
+            "server",
+            "linux"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original"
```
Comment (Panquesito7) [inline devicon.json:531]:
```suggestion
                "original",
                "original-wordmark"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 5]
```diff
@@ -516,6 +516,34 @@
...
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#932178",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
```
Comment (Panquesito7) [inline devicon.json:540]:
```suggestion
            },
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Haha, this is the bug that the Peek Icons bot reported 😅 it was kind of hidden.

**Review by Panquesito7 — DISMISSED** [review]
Great work, @MichielPater; LGTM! Thank you for your contributions and time in adding new icons to our repository! 😄🎉

**Comment by Thomas-Boi** [comment]
Hi @MichielPater,

I'm sorry for the late response 😅. I've been distracted by a few other tasks and life in general.

Unfortunately, the icon not clear enough that it's `centos` since it now looks like a black geometric figure, which can be any other brands.

Perhaps we can make the outlines of the arrows transparent like the center? I think this will show the arrows and make it clearer that this is `centos`. If you can take the original version and make those outlines transparent, I think that'd be best.
image

Let us know if you are up to this 😄

File: icons/centos/centos-plain.svg [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+</svg>
```
Comment (Panquesito7) [inline icons/centos/centos-plain.svg:37]:
```suggestion
<svg width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="m64.633 5.723-12.62 12.863h8.444v33.818l4.176 4.258 4.13-4.207V18.811h8.71L64.633 5.723zM22.37 21.355l.002 22.903 9.266-9.246L53.633 56.97h4.105v-3.418l-21.92-22.358 9.66-9.84H22.372zm26.938 0-9.65 9.838 18.08 18.44V21.355h-8.43zm22.177.215-.002 28.108L89.61 31.195l-9.43-9.625h-8.695zm13.395.883 8.566 8.742-21.96 22.399v3.597h3.933l21.855-22.29 9.438 9.62V22.453H84.88zm12.394 16.361-18.02 18.377h27.459V48.43l-9.438-9.616zm-65.632.069-9.272 9.256v8.832h27.39L31.643 38.883zM19.654 50.846 6.28 64.196 19.654 77.55v-9.328H52.98l4.03-4.028-4.49-4.472H19.655v-8.877zm89.778.357-.004 8.746H76.55l-4.25 4.328 3.86 3.942h33.038v9.373l13.057-13.315-12.824-13.074zM22.377 70.977l.002 9.287 9.264 9.256 18.572-18.543H22.377zm31.713 0L31.639 93.383l-9.262-9.248v22.677h22.21l-9.544-9.527 22.695-22.65v-3.658H54.09zm17.256 0v4.175L92.56 96.316l.964.973-8.61 8.596h21.562v-21.61l-8.24 8.414-.962.967-22.242-22.68h-3.687zm7.517 0 18.412 18.767 9.202-9.379v-9.388H78.863zm-14.576.998-3.822 3.814v33.787h-9.233l13.055 13.028 13.057-13.028h-8.717V76.31l-4.34-4.334zm-6.547 6.533-18.816 18.78 9.547 9.526h9.27V78.508zm13.606.512v27.8h8.756l9.55-9.533L71.346 79.02z"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
These suggestions optimize the SVGs. 🙂

**Review by Panquesito7 — APPROVED** [review]

**Comment by Thomas-Boi** [comment]
Wow @MichielPater

It looks great now! Thank you very much for your effort 😄

Just so you know, we are temporarily pausing accepting icons PR. We are upgrading our peek-bot and we want to ensure this PR matches our new standards. The upgrade will be done by the next build release, which should be done either this or next week. 

Thank you for your patient and commitment!

**Review by Thomas-Boi — APPROVED** [review]
Everything looks good! I still can't get over how good centos looks now. Great job @MichielPater

**Review by Panquesito7 — APPROVED** [review]
Nice work on all your contributions, @MichielPater! Thanks! 😄👍🎉

**Pull request merged**